src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -214,7 +241,7 @@ class GitOpsConfiguration extends Component<GitOpsProps, GitOpsState> {
         return {
             host: this.requiredFieldCheck(form.host),
             username: this.requiredFieldCheck(form.username),
-            token: this.requiredFieldCheck(form.token),
+            token: this.state.form.id ? '' : this.requiredFieldCheck(form.token),
```

Comment:
Can reduce ternary operator everywhere `this.state.form.id && this.requiredFieldCheck(form.token)`